airflow/api_connexion/openapi/v1.yaml

```
@@ -478,6 +478,23 @@ paths:
         '404':
           $ref: '#/components/responses/NotFound'
 
+    delete:
+      summary: Delete a DAG
```

```suggestion
      summary: Delete a DAG
      description: This endpoint will delete ALL metadata, DAGRuns, etc EXCEPT Log. This action cannot be undone
```
I think that we should add this description as we do in the UI, so users are aware of what this can cause